test/lib/clients/repositories_test.rb

```diff
@@ -105,4 +105,46 @@
       end
     end
   end
+
+  describe "#SortReposByPriority" do
+    it "returns nil if param is nil" do
+      expect(client.SortReposByPriority(nil)).to eq nil
+    end
+
+    it "returns input sort by priorities" do
```

Comment:
Use `it "sorts by priorities"`, sounds better and is consistent with the other test below.